README.md

```diff
@@ -153,6 +153,21 @@ PATH += ~/.nvs/default/bin
 ```
 *This feature is not available in Windows Command Prompt. Use PowerShell instead.*
 
+<a name="nvmrcNote"><sup>1</sup></a> *`.nvmrc` parsing is best effort at the moment. Please log an issue if `nvs` fails on a valid `.nvmrc` file.*
```

Comment:
Let me know if this looks good or if you'd prefer it re-worded!

Reply:
I think this note is completely unnecessary. We're confident that the `.nvmrc` parsing will work for all common cases. (And frankly entire tool is "best effort".)